### PR TITLE
Added release notes for CDN Release - 20160922

### PR DESCRIPTION
**Pull request by amitgandhinz**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview is now ready:

* [`internal-api-docs`](https://staging.developer.rackspace.com/build-a481ef71b9/docs/cdn/v1/)
* [`api-docs`](https://staging.developer.rackspace.com/build-a481ef71b9/docs/cdn/v1/)

File: api-docs/release-notes/releases/cdn-v1-20160922.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+What's new
+----------
+
+|no changes|
+
+Resolved issues
+---------------
+
+- Validation added to Top Level Domains during Secure Certificate requests
```
Comment (cyrichardson) [inline api-docs/release-notes/releases/cdn-v1-20160922.rst]: Validation added to top level domains during secure certificate requests.

**Comment by cyrichardson** [comment]
@amitgandhinz 1 comment on first bullet

**Comment by rackernexus** [comment]
Your content preview is now ready:

* [`internal-api-docs`](https://staging.developer.rackspace.com/build-2366233d8a/docs/cdn/v1/)
* [`api-docs`](https://staging.developer.rackspace.com/build-2366233d8a/docs/cdn/v1/)

File: api-docs/release-notes/releases/cdn-v1-20160922.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+What's new
+----------
+
+|no changes|
+
+Resolved issues
+---------------
+
+- Validation added to top level domains during secure certificate requests
```
Comment (cyrichardson) [inline api-docs/release-notes/releases/cdn-v1-20160922.rst]: Add period (.) at end.

**Comment by rackernexus** [comment]
HULK PREVIEW

* [`internal-api-docs`](https://staging.developer.rackspace.com/build-3b59f21580/docs/cdn/v1/)
* [`api-docs`](https://staging.developer.rackspace.com/build-3b59f21580/docs/cdn/v1/)

**Comment by cyrichardson** [comment]
LGTM - waiting for deploy to complete. Will the merge/push out to developer.rackspace.com.

**Pull request merged**